### ISSUE TITLE
Materials' Bottom row is not scanned properly

### ISSUE THREAD

**Issue by Rafii2198** [Andrewthe13th/Inventory_Kamera#223]
**V1.2.2**
This is image from logging folder:
![CharacterDevelopmentItemsInventory2](https://user-images.githubusercontent.com/26695327/154777156-f2d7dca7-023b-457d-b834-040b9aaef16e.png)

As you can see, the bottom row of items is a bit faded, but it tries to scan them failing most of them time.
This also happens with character development items as it's pretty much same gui.

**Comment by Cupcak3** [comment]
Hmm, the scanner should not be trying to click through at all the bottom row of a page of items specifically because of the fading you described. Does that bottom row of items appear as the first row of items when the page is scrolled?

**Comment by Rafii2198** [comment]
No, it scanned them and failed with most of them but not all and just skipped it. Just to make sure I imported materials to Seelie and there were always some missing records. I also noticed it fails to scan on regular item, I made few scans and it all seams like one big RNG, what will fail, which row will be faded, it just changes with each of my scans, and on each of these scans there were few random items that were missing as well some bigger batches because of that fading.
